src/services/amqOnlineServices.js

```diff
@@ -139,24 +139,32 @@ const provisionAddressSpace = (username, namespace) => {
   return findOrCreateOpenshiftResource(addressSpaceDef(namespace), asRes);
 };
 
-const handleAddressSpaceUpdateEvent = (username, dispatch, resolve, event) => {
+const handleAddressSpaceUpdateEvent = (username, namespace, dispatch, resolve, event) => {
   if (!event || !event.status || !event.status.isReady) {
-    dispatch(getPayloadFromAddressSpace(event));
+    dispatch(getPayloadFromAddressSpace(event, username, namespace));
     return;
   }
   const messagingEndpointDef = event.status.endpointStatuses.find(e => e.name === 'messaging');
   if (!messagingEndpointDef) {
     return;
   }
-  const payload = getPayloadFromAddressSpace(event);
+  const payload = getPayloadFromAddressSpace(event, username, namespace);
   payload.additionalAttributes = Object.assign({}, payload.additionalAttributes, {
     'enmasse-credentials-username': cleanUsername(username),
     'enmasse-credentials-password': window.atob(genEvalPassword(username))
   });
   resolve(payload);
 };
 
-const getPayloadFromAddressSpace = as => {
+const buildAMQOnlineConcoleUrl = (username, namespace) => {
```

Comment:
Should this be called `username`? Is this not the AddressSpace name?

Reply:
The address space is named after the logged in user

Reply:
perfect, thanks @pb82